index.rst

```diff
@@ -145,7 +146,7 @@ Quickstart
 ::
 
     [root@docker tests]# cd docker
-    [root@docker docker]# git checkout $(git tag --list | tail -1)
+    [root@docker tests]# git reset origin/master
```

Comment:
This is right on the heels of a git clone. Why not just remove this line entirely?